### PR TITLE
ShardingSphere plugins conflict among themselves.

### PR DESCRIPTION
**Pull request by yu199195**
…efine.

Please answer these questions before submitting pull request

- Why submit this pull request?
- [x] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues
#4920 
___
### Bug fix
- Bug description.
  have some same class define will find other interceptor
- How to fix?
 add witness class to fix.

### CODE REVIEW

File: test/plugin/scenarios/shardingsphere-4.x-scenario/support-version.list [hunk 1]
```diff
@@ -14,7 +14,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-4.0.0
-4.0.1
 4.1.0
-4.1.1
```
Comment (wu-sheng) [inline test/plugin/scenarios/shardingsphere-4.x-scenario/support-version.list:20]: Why remove other versions?

File: .github/workflows/plugins-test.2.yaml [hunk 2]
```diff
@@ -42,6 +42,10 @@ jobs:
           - { name: 'resttemplate-4.x-scenario', title: 'RestTemplate 4.0.0.RELEASE-4.3.26.RELEASE (57)' }
           - { name: 'servicecomb-0.x-scenario', title: 'ServiceComb 0.x (5)' }
           - { name: 'servicecomb-1.x-scenario', title: 'ServiceComb 1.x (5)' }
+          - { name: 'shardingsphere-3.x-scenario', title: 'ShardingSphere 3.0.0 (1)' }
+          - { name: 'shardingsphere-4.x-RC1-RC2-scenario', title: 'ShardingSphere 4.0.0-RC1-4.0.0-RC2 (2)' }
+          - { name: 'shardingsphere-4.x-RC3-scenario', title: 'ShardingSphere 4.0.0-RC3 (1)' }
+          - { name: 'shardingsphere-4.x-scenario', title: 'ShardingSphere 4.1.0 (1)' }
```
Comment (wu-sheng) [inline .github/workflows/plugins-test.2.yaml:48]: Whether is this plugin for 4.x or 4.1.0 only?

Comment (yu199195) [reply .github/workflows/plugins-test.2.yaml:48]: The name is a bit confusing, initially we wanted to support 4.0.0 to 4.1.1.
now it noly supprot 4.1.X

File: test/plugin/scenarios/shardingsphere-4.x-scenario/support-version.list (hunk 1 above)
Comment (yu199195) [reply test/plugin/scenarios/shardingsphere-4.x-scenario/support-version.list:20]: add 4.1.1

Comment (yu199195) [reply test/plugin/scenarios/shardingsphere-4.x-scenario/support-version.list:20]: already add 4.1.1

**Pull request merged**